transcom/mymove
# B-18174 Added ppm pro gear weight ticket to history

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A871868&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

PPM Processor has made updates to pro-gear weights and then made a decision to Accept or Reject the requested expenses.

### How to test

1. Access the office app
2. Login as a service counselor
3. Open a PPM in document review
4. Review a Pro gear and modify the progear/spouse progear weights
5. Check Move History for Update Pro-gear records

### Frontend
